ci/.github/workflows/ci.yml

```
@@ -27,7 +27,7 @@ jobs:
           - 2.3
         env:
           -
-            DIFF_LCS_VERSION: "> 1.4.3"
+            BOGUS: ""
```

As I said in its originating PR, this needs fixing, `BOGUS` is not clear or informative. 
```suggestion
            __UNUSED_VARIABLE: "Prevents build error when {matrix.env} is used."
```